src/plugins/data/server/search/routes/search.ts

```diff
@@ -36,6 +36,7 @@ export function registerSearchRoute(router: DataPluginRouter): void {
                 sessionId: schema.maybe(schema.string()),
                 isStored: schema.maybe(schema.boolean()),
                 isRestore: schema.maybe(schema.boolean()),
+                retrieveResults: schema.maybe(schema.boolean()),
```

Comment:
how is this related ?

Reply:
This parameter was added in https://github.com/elastic/kibana/pull/179679/files#diff-397d729961b0223d5105823a61edbaa64f27572c2235c3d2582f89bb57b69ad7 but only worked when bfetch was enabled. As soon as you disabled bfetch (and used http routes), this parameter wasn't actually getting forwarded to the search strategy.